packages/orbit-components/src/Card/README.md

```diff
@@ -49,23 +49,22 @@ import Card, { CardSection } from "@kiwicom/orbit-components/lib/Card";
-| initialExpanded | `boolean`                  | `false` | Initial state of expandable CardSection when it mounts in uncontrolled variant. Can only be used if `expandable` is `true`.                                                                             |
-| noSeparator     | `Boolean`                  |         | Optional prop to turn off Separator between `header` and `children`                                                                                                                                     |
-| onClick         | `event => void \| Promise` |         | Function for handling the onClick event.                                                                                                                                                                |
-|                 |
-| onClose         | `() => void \| Promise`    |         | Callback that is triggered when section is closing                                                                                                                                                      |
-| onExpand        | `() => void \| Promise`    |         | Callback that is triggered when section is expanding                                                                                                                                                    |
-| title           | `React.Node`               |         | The title of the CardSection                                                                                                                                                                            |
-| titleAs         | [`enum`](#enum)            | `"h2"`  | The element used for the root node of the title of CardSection.                                                                                                                                         |
+| Name            | Type                       | Default | Description                                                                                                                                                                                                                                 |
+| :-------------- | :------------------------- | :------ | :------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| actions         | `React.Node`               |         | Actions which will be rendered on the right side. If used with `expandable`, the rendered elements should not be interactive but can be used to represent the expanded state.                                                               |
```

Comment:
Shouldn't we mention the similar for usage of `action` together with `onClick`? 🤔